tools/eslint/src/check-license.js

```diff
@@ -0,0 +1,69 @@
+/**
```

Comment:
Let's use TypeScript like `extensions.ts` or `use-using.ts`, to get better typing.